raml/abao_hooks.js

```diff
@@ -85,3 +85,81 @@ hooks.before('GET /api/{version}/status/performance/{period} -> 200', function (
   test.request.params = params;
   done();
 });
+
+// runbooks
+var correct_runbook = {description: "test", name: "test", runbook: "test", type: "test"}
```

Comment:
I would recommend to use same naming style within the file - mixedCase (like at L4-7):

    var bodyRunbook = { ... }  // correct runbook
    var bodyBadRunbook = { ... }